### DISCUSSION TITLE
Vim options lua

### DISCUSSION THREAD

**Discussion by josephcrawfordSRH** [LazyVim/LazyVim#2742] [Q&A] [answered]
How can i set nobackup in vim when using lazyvim and init.lua?

I have tried to following but it does not seem to fix my problem.
```
vim.g.backup = false
vim.g.writebackup = false
```
I have also tried vim.opt.backup etc.

The issue I am having is that file watchers are removing the file from the list due to the way nvim is using backups as it will write a file remove the original and write the new file.

I need to have my watchers working with nvim 

For more context I first reached out to the Deno project: https://github.com/denoland/deno/issues/22901#issuecomment-1997179875

**Comment by dpetka2001** [comment, marked as answer]
You should set these options in your `~/.config/nvim/lua/config/options.lua`. By default `backup` is set to `nobackup`, so you should only set the other option like `vim.opt.writebackup = false`. You can see what the default options are set in Neovim by doing `:set backup?` and `:set writebackup?`. And in general `:set option_name?`. The questionmark is not a typo. You actually need to type it, so that Neovim shows you the default value.

**Reply by josephcrawfordSRH** [reply]
Thank you.  I will try this Monday morning. Hopefully it solves my issue with he watcher.
